packages/manager/tools/component-rollup-config/src/plugins/translation-utils.ts

```diff
@@ -0,0 +1,113 @@
+import concat from 'lodash/concat';
+import fs from 'fs';
+import path from 'path';
+import slash from 'slash';
+import startsWith from 'lodash/startsWith';
+
+const ALL_LANGUAGES = [
+  'cs_CZ',
+  'de_DE',
+  'en_ASIA',
+  'en_AU',
+  'en_CA',
+  'en_GB',
+  'en_SG',
+  'en_US',
+  'es_ES',
+  'es_US',
+  'fi_FI',
+  'fr_CA',
+  'fr_FR',
+  'it_IT',
+  'lt_LT',
+  'nl_NL',
+  'pl_PL',
+  'pt_PT',
+];
```

Comment:
Does it make sense to have a `module` made of constants ? I think there's also a list in `manager-core`